Source/m_misc.c

```diff
@@ -1961,19 +1965,36 @@ default_t defaults[] = {
 
   // [FG] music backend
   {
-    "music_backend",
-    (config_t *) &music_backend, NULL,
-    {0}, {0, num_music_backends-1}, number, ss_none, wad_no,
+    "midi_player",
+    (config_t *) &midi_player, NULL,
+    {0}, {0, num_midi_players-1}, number, ss_none, wad_no,
     "0 for SDL2_Mixer (default), 1 for OPL Emulation"
   },
 
+#if defined(HAVE_FLUIDSYNTH)
   {
-    "fluidsynth_sf_path",
-    (config_t *) &fluidsynth_sf_path, NULL,
-    {SFINIT(.s, "")}, {0}, string, ss_none, wad_no,
+    "soundfont_path",
+    (config_t *) &soundfont_path, NULL,
+    {SFINIT(.s, "soundfonts"DIR_SEPARATOR_S"TimGM6mb.sf2")},
+    {0}, string, ss_none, wad_no,
     "FluidSynth soundfont path"
   },
 
+  {
```

Comment:
Do we really need to set chorus and reverb? Should we just keep them disabled?

Reply:
Personally, I don't like reverb and chorus, but maybe someone needs it. PrBoom+ and GZDoom can enable it.

Reply:
Okay, I don't like them either, that's why I am asking. 😄 But I agree it won't hurt to leave them as a choice.